.fussyfox.yml

```diff
@@ -1,3 +1,3 @@
 # - bandit
```

Comment:
Can we uncomment this now?

Reply:
Sure, I just saw that I uncommented the wrong line from the file. Sorry